lib/datastax/cloud/index.js

```diff
@@ -198,6 +198,17 @@ function getMetadataServiceInfo(cloudOptions, callback) {
 
 const getMetadataServiceInfoAsync = util.promisify(getMetadataServiceInfo);
 
+async function refreshMetadataServiceInfo(options) {
```

Comment:
This function is now poorly named.  It also is equivalent to datastax.cloud.init() if we exclude the SNI special case resolution logic.  It might make more sense to just make that function more general.